tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -284,24 +284,58 @@ async def started(iens):
     assert was_started == started_expected
 
 
+@given(words, st.integers(min_value=1), words)
+@pytest.mark.usefixtures("capturing_qsub")
+async def test_full_resource_string(memory_per_job, num_cpu, cluster_label):
+    driver = OpenPBSDriver(
+        memory_per_job=memory_per_job if memory_per_job else None,
+        cluster_label=cluster_label if cluster_label else None,
+    )
+    await driver.submit(0, "sleep", num_cpu=num_cpu)
```

Comment:
Do we have to submit here, or can we just unit test the `parse_resource_string` function?

Reply:
parse_resources_string depends on the qsub command having run, so yes.

Reply:
Ohhh, I thought this was the `build_resource_string` function